alpha/lib/model/MediaServerNode.php

```diff
@@ -109,4 +112,14 @@ public static function modifyUrlForVodFromLive($liveUrl, DeliveryProfileDynamicA
 		return $liveUrl;
 	}
 
+	public static function getExplicitLiveUrl($liveUrl, LiveStreamEntry $entry)
+	{
+		$userType = static::USER_TYPE_ADMIN;
```

Comment:
use self and not static

Reply:
We must use static as we want to take this value from the inheritance, self will cause it to take it from the parent (current) class - which isn't the value we want